documentation/releases/downloads.md

```diff
@@ -24,14 +24,14 @@ Adding the gradle dependency in a non-multiplatform project:
 ```groovy
```

Comment:
Hi @zsmb13 thanks for the PR! Can you please change the language from groovy to kotlin in the places where you added parentheses?

Reply:
Updated the language. I didn't do it earlier because I thought it must be Groovy because that's what the docs website has highlighting for.

Reply:
![image](https://github.com/bumble-tech/appyx/assets/12054216/5d07b4e1-cc21-4fcf-ab7f-3333faa62152)

Checked it locally, looks all good